gui/default/syncthing/core/syncthingController.js

```diff
@@ -928,6 +928,9 @@ angular.module('syncthing.core')
         $scope.deviceStatus = function (deviceCfg) {
             var status = '';
 
+            if( deviceCfg.deviceID === this.myID){
+                return '';
+            }
```

Comment:
What case is this trying to address?

Reply:
The current device always shows as discounted. Maybe this is why disconnected was not included.